src/Creator/ApiKeysTestCreator.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusMolliePlugin\Creator;
+
+use BitBag\SyliusMolliePlugin\Client\MollieApiClient;
+use BitBag\SyliusMolliePlugin\DTO\ApiKeyTest;
+use Mollie\Api\Resources\MethodCollection;
+
+final class ApiKeysTestCreator implements ApiKeysTestCreatorInterface
+{
+    /** @var MollieApiClient */
```

Comment:
interface?

Reply:
no interface in MollieApiClient :)

Reply:
it would be nice to add an interface :)

Reply:
Yes true but base MollieApiClient from mollie packages don't have an interface so if I create myself can cause problems